homeworks/roman.pavliuk_RomanPvlk/5-logic/homework.js

```diff
@@ -0,0 +1,34 @@
+function calculate(x, y, operation) {
+    let result = 0;
+    const numberX = parseFloat(x);
```

Comment:
This implementation makes possibly follow input. It's ok for study projects, but for real ones it would be treated as a bug. 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/3ecdc0bf-ddf6-4275-8dfc-c94073e76028)

The easiest solution here is to set `type="number" to the inputs

Reply:
I will try to fix it too next time